docs/src/modules/java/nav.adoc

```diff
@@ -8,8 +8,7 @@
 *** xref:java:replicated-entity.adoc[Implementing Replicated Entities]
 *** xref:java:actions.adoc[Implementing Actions]
 *** xref:java:actions-as-controller.adoc[Actions as Controllers]
-*** xref:java:entity-eventing.adoc[Subscribing to a Journal]
-*** xref:java:topic-eventing.adoc[Publishing and Subscribing]
+*** xref:java:actions-publishing-subscribing.adoc[Publishing and Subscribing with Actions]
```

Comment:
No problems with that being a tad long in the left nav menu when rendered?

Reply:
We get a break line, that's all. But happy to change if we find another option. 

Maybe only "Publishing and Subscribing" ?

Reply:
```suggestion
*** xref:java:actions-publishing-subscribing.adoc[Publishing and Subscribing]
```